generators/app/constants.js

```diff
@@ -90,13 +107,16 @@ module.exports.BOOTSTRAP_TYPES = [
 ];
 
 module.exports.DEV_DEPENDENCIES = [
-  'eslint-plugin-import@2.18.2',
+  'eslint-plugin-import@2.22.1',
   'eslint-plugin-jsx-a11y@6.2.3',
   'eslint-plugin-prettier@3.1.0',
   'eslint-plugin-react@7.14.3',
+  // TODO: Do we need this?
```

Comment:
https://github.com/Wolox/eslint-config-wolox-react/blob/master/index.js

It's being used, I think it as something related to optional chaining. In any case, to remove it we'd have to change the linter